### ISSUE TITLE
Wrapping of lines needs fixing to avoid horizontal scrolling

### ISSUE THREAD

**Issue by SusanBrown** [Thaerious/NERVE#96]
![image](https://cloud.githubusercontent.com/assets/545952/15931204/b9ce188e-2e24-11e6-9aac-8e2f57c57257.png)

**Comment by abramoffcwrc** [comment]
I agree with this. This needs to be fixed in order for the program to be used.
